api/v1/version.go

```diff
@@ -176,11 +178,20 @@ func (v *VerticaDB) IsUpgradePathSupported(newAnnotations map[string]string) (ok
 	return
 }
 
-// isOnlineUpgradeSupported returns true if the version in the Vdb is for a version that supports online upgrade
-// 24.3.0-4 and 24.4.0-1 are the oldest version that support online upgrade, anything newer is fine
+// isOnlineUpgradeSupported returns true if the version in the Vdb is is equal or newer than
+// 24.3.0-2.
 func (v *VerticaDB) isOnlineUpgradeSupported(vinf *version.Info) bool {
-	if vinf.IsEqualOrNewer(OnlineUpgradeVersion244) {
-		return vinf.IsEqualOrNewerWithHotfix(OnlineUpgradeVersion244)
+	return vinf.IsEqualOrNewerWithHotfix(OnlineUpgradeVersion)
+}
+
+// IsPausedSessionsSupported returns true if the vertica version supports the expected pause sessions semantics
+func (v *VerticaDB) IsPausedSessionsSupported() bool {
+	vinf, ok := v.MakeVersionInfo()
+	if !ok {
+		return false
+	}
+	if vinf.IsEqualOrNewer(MinPauseSessionsVersion243) {
+		return vinf.IsEqualOrNewerWithHotfix(MinPauseSessionsVersion243)
 	}
-	return vinf.IsEqualOrNewerWithHotfix(OnlineUpgradeVersion243)
+	return vinf.IsEqualOrNewerWithHotfix(MinPauseSessionsVersion243)
```

Comment:
You need to update this function. You never use `MinPauseSessionsVersion244`.
Is it easier to just exclude versions  that do not support paused sessions(lower than v24.3.0-4 and equal to v24.4.0-0)?